packages/babel-helper-remap-async-to-generator/src/index.js

```diff
@@ -14,11 +14,24 @@ let buildWrapper = template(`
   })
 `);
 
+let arrowBuildWrapper =  template(`
+  (() => {
+    var ref = FUNCTION;
+    return (PARAMS) => ref.apply(this, arguments);
```

Comment:
Maybe need the test option. something like "evaluated": true|false that will exec compilation result. Then some assertions can be checked by runtime.